internal/gardener/gardener.go

```diff
@@ -89,7 +89,7 @@ func (g *gardenerProvisioner) Credentials(cluster *types.Cluster, provider *type
 		return nil, err
 	}
 
-	s, err := k8s.CoreV1().Secrets(fmt.Sprintf("garden-%s", provider.ProjectName)).Get(fmt.Sprintf("%s.kubeconfig", cluster.Name), metav1.GetOptions{})
+	s, err := k8s.CoreV1().Secrets(provider.ProjectName).Get(fmt.Sprintf("%s.kubeconfig", cluster.Name), metav1.GetOptions{})
```

Comment:
The previous approach was already much better from a UX perspective, we can't assume that the users have to know they have to prefix their project name with "garden-"

Reply:
reverted back to old approach